README.md

```diff
@@ -22,6 +22,35 @@ A detailed tutorial is provided on our [website](https://federatedscope.io/).
 - [05-25-2022] Our paper [FederatedScope-GNN](https://arxiv.org/abs/2204.05562) has been accepted by KDD'2022!
 - [05-06-2022] We release FederatedScope v0.1.0! 
 
+## Code Structure
+```
+FederatedScope
+├── federatedscope
+│   ├── core           
+│   |   ├── worker               # Behaviors of participants (i.e., server and clients)
+│   |   ├── configs              # Customizable configurations
+│   |   ├── trainer              # Detail of local training
+│   |   ├── monitors             # The monitor module for logging and demonstrating  
```

Comment:
We should unify the plural of the words here (since we have `worker/trainer` rather than `workers/trainers`)

Reply:
Modify to `workers/trainers`